src/codemagic/apple/app_store_connect/resource_manager.py

```diff
@@ -74,6 +78,10 @@ def __init__(self, client: AppStoreConnectApiClient):
     def resource_type(self) -> Type[R]:
         raise NotImplemented
 
+    @classmethod
+    def _get_include_field_name(cls, include_type: Type[R]) -> str:
+        raise NotImplementedError()
```

Comment:
```suggestion
        raise NotImplemented
```
suits here better. `NotImplementedError` will trigger IDEs and linters to complain that classes which inherit from `Resource` should implement this abstract method, which is not the case.